Automattic/vip-go-mu-plugins
# Update/ensure index

## Description


## Changelog Description


### Plugin Updated: Search

This change ensures the index is correctly initialized in ElasticSearch before operations that could create a default index. Default index doesn't have the correct mapping set and therefore could potentially cause search issues.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Replace index and ES cluster in the following:
1. Delete index `curl -X DELETE http://127.0.0.1:49155/vip-200508-post-1`
2. Add a post `dev-env wp -- post create --post_title="Test" --post_content="Content" --post_excerpt="Test" --post_status="publish"`
3. Check post_date `type` to be `date` - `curl http://127.0.0.1:49155/vip-200508-post-1/_mapping | jq | grep -A 5 post_date`
